google/cloud/pubsub_v1/publisher/flow_controller.py

```diff
@@ -285,7 +285,9 @@ def _load_info(self, message_count=None, total_bytes=None, reserved_bytes=None):
         if total_bytes is None:
             total_bytes = self._total_bytes
 
-        if reserved_bytes is None:
+        # The following check is here for completness even though it always passes (no
+        # calling code currently needs to override reserved_bytes).
+        if reserved_bytes is None:  # pragma: no branch
```

Comment:
Why not just remove the argument?  YAGNI?

Reply:
On a second thought, yeah, why keep this around. Will remove it. :+1: